.github/CODEOWNERS

```diff
@@ -30,813 +30,810 @@
 # PRLabel: %azure-spring
 /sdk/boms/spring-cloud-azure-dependencies/           @saragluna @chenrujun @netyyyy @moarychan
 
-####
-# Service teams
-####
 
-# AzureSdkOwners: @srnagar
-# ServiceLabel: %graalvm
-# PRLabel: %graalvm
-/sdk/aot/                                            @srnagar @jonathangiles
+# ######## Core Libraries ########
 
-# AzureSdkOwners: @mssfang
-# ServiceLabel: %App Configuration
-# PRLabel: %App Configuration
-/sdk/appconfiguration/                               @mssfang @alzimmermsft
+# AzureSdkOwners: @alzimmermsft
+# ServiceLabel: %Azure.Core
+# PRLabel: %Azure.Core
```

Comment:
There must be a space between these.  If `#ServiceLabel` is not in its own block, it will assume this is a service-owned library and that the people in the path are "Service Attention" members.